### PR TITLE
add lauch file for example

### PR DESCRIPTION
**Pull request by pietrocolombo**
To make people understand how to use it

### CODE REVIEW

File: imu_filter_madgwick/launch/imu_filter_madgwick.launch [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  #### IMU Driver ###########################################################
+
+  <node pkg="nodelet" type="nodelet" name="ImuFilterNodelet" 
+    args="load imu_filter_madgwick/ImuFilterNodelet imu_manager" 
+    output="screen">
+
+    # supported data rates: 4 8 16 24 32 40 ... 1000 (in ms)
+    <param name="use_magnetic_field_msg" value="true"/>
+
```
Comment (mintar) [inline imu_filter_madgwick/launch/imu_filter_madgwick.launch]: The parameter use_magnetic_field_msg was removed long time ago, so this line should be removed as well.

File: imu_filter_madgwick/launch/imu_filter_madgwick.launch [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <param name="fixed_frame" value="map"/>
+    <param name="publish_tf" value="false"/>
+
+    <remap from="/imu/data_raw" to="/piksi/imu"/>
+    <remap from="/imu/mag" to="/piksi/mag"/>
+
+  </node>
+  
+
+
```
Comment (mintar) [inline imu_filter_madgwick/launch/imu_filter_madgwick.launch]: Please delete the extra empty lines.

File: imu_filter_madgwick/launch/imu_filter_madgwick.launch [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  #### IMU Driver ###########################################################
+
+  <node pkg="nodelet" type="nodelet" name="ImuFilterNodelet" 
+    args="load imu_filter_madgwick/ImuFilterNodelet imu_manager" 
+    output="screen">
+
+    # supported data rates: 4 8 16 24 32 40 ... 1000 (in ms)
+    <param name="use_magnetic_field_msg" value="true"/>
+
+    <param name="fixed_frame" value="map"/>
```
Comment (mintar) [inline imu_filter_madgwick/launch/imu_filter_madgwick.launch]: I think this line should be removed as well.

File: imu_filter_madgwick/launch/imu_filter_madgwick.launch [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    <param name="publish_tf" value="false"/>
+
+    <remap from="/imu/data_raw" to="/piksi/imu"/>
+    <remap from="/imu/mag" to="/piksi/mag"/>
```
Comment (mintar) [inline imu_filter_madgwick/launch/imu_filter_madgwick.launch]: These remappings are specific to your IMU (piksi), so please remove them.

**Review by mintar — CHANGES_REQUESTED** [review]
Thanks for your PR! I've added some comments.

Also: The "indigo" branch is very outdated. Please target your PR against the "noetic" branch.

**Review by pietrocolombo — COMMENTED** [review]
ok that's fine, but I can't switch branches for merge

**Comment by mintar** [comment]
You deleted a bit too much. Please re-insert this line:

```xml
     <param name="publish_tf" value="false"/>
```

About changing the base branch: Click "Edit" on your PR, then **below** the title, you can see something like: `pietrocolombo wants to merge 2 commits into...`; here, select `noetic`.

If it's too difficult, just open a new pull request.

**Comment by mintar** [comment]
Oh, I see you already opened a new PR. I'll close this one.

**Pull request closed without merging**